src/main/java/pinger/Pinger.java

```diff
@@ -18,19 +18,24 @@ public static void main(String[] args)  {
                 String urlI = "http://" + reader.readLine();
                 URL url = new URL(urlI);
                 HttpURLConnection conn = (HttpURLConnection) url.openConnection();
```

Comment:
You also need to remove the cast here. Change this to a `URLConnection` and create another variable inside the `if` statement which does the cast.